clusterloader2/testing/density/100_nodes/override.yaml

```diff
@@ -1 +1,2 @@
 DENSITY_RESOURCE_CONSTRAINTS_FILE: ./100_nodes/constraints.yaml
+LATENCY_POD_MEMORY: 350
```

Comment:
Why can't we change it everywhere? What's the point of introducing differences between tests?

Reply:
You're right. I'm changing density test default.